src/c_api.cpp

```diff
@@ -921,7 +921,7 @@ int LGBM_DatasetCreateFromSampledColumn(double** sample_data,
     omp_set_num_threads(config.num_threads);
   }
   DatasetLoader loader(config, nullptr, 1, nullptr);
-  *out = loader.CostructFromSampleData(sample_data, sample_indices, ncol, num_per_col,
+  *out = loader.ConstructFromSampleData(sample_data, sample_indices, ncol, num_per_col,
                                        num_sample_row,
                                        static_cast<data_size_t>(num_total_row));
```

Comment:
There are more chars in the new name, so all indents for the parameters should be updated:

![image](https://user-images.githubusercontent.com/25141164/88667847-8aee0200-d0ea-11ea-91c4-8224e75a46c0.png)